core/parameters/src/config_store.rs

```diff
@@ -105,6 +107,24 @@ impl RuntimeConfigStore {
                 store.insert(*protocol_version, Arc::new(runtime_config));
             }
         }
+        if is_benchmarknet {
+            let protocol_version = 69;
+            let diff_bytes = BENCHMARKNET_DIFF;
+            let diff :ParameterTableDiff= diff_bytes.parse().unwrap_or_else(|err| panic!("Failed parsing runtime parameters diff for version {protocol_version}. Error: {err}"));
+            params.apply_diff(diff).unwrap_or_else(|err| panic!("Failed applying diff to `RuntimeConfig` for version {protocol_version}. Error: {err}"));
+            #[cfg(not(feature = "calimero_zero_storage"))]
+            store.insert(
+                protocol_version,
+                Arc::new(RuntimeConfig::new(&params).unwrap_or_else(|err| panic!("Failed generating `RuntimeConfig` from parameters for version {protocol_version}. Error: {err}"))),
+            );
+            #[cfg(feature = "calimero_zero_storage")]
```

Comment:
We don't need to support "benchmarknet" on Calimero, as it will never be used in that context, so we can get rid of this branch.